client/src/components/ReviewEntry.jsx

```diff
@@ -2,18 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import {
-  Wrapper,
-  ReviewUser,
-  ReviewDate,
-  ReviewText,
+  ReviewEntryWrapper,
+  ReviewUserWrapper,
+  ReviewDateWrapper,
+  ReviewTextWrapper,
 } from './styles/ReviewEntry.styles';
 
 const ReviewEntry = props => (
-  <Wrapper>
-    <ReviewUser> {props.review.first_name} </ReviewUser>
-    <ReviewDate> {props.review.review_date} </ReviewDate>
-    <ReviewText> {props.review.review_text} </ReviewText>
-  </Wrapper>
+  <ReviewEntryWrapper>
```

Comment:
These new styled component names are much more descriptive

Reply:
Thank you !  Longer for sure..